src/common/Circle.cpp

```diff
@@ -7,9 +7,8 @@
 
 Circle::Circle(float size, sf::Vector2f position):mCircleShape(size) {
     mCircleShape.setPosition(position);
-    unsigned int seed = time(NULL);
-    mCircleShape.setFillColor(
-        sf::Color(rand_r(&seed) % 255, rand_r(&seed) % 255, rand_r(&seed) % 255));
+    mColor = sf::Color(0, 0, 255);
```

Comment:
mCircleShape has a color property, why dont you just uses that to store the color
https://www.sfml-dev.org/tutorials/2.0/graphics-shape.php

Reply:
good point, looks like I can call `getFillColor` to grab it

Reply:
Yeah let's use that so that we use less variables/code

Reply:
fixed in 92f3e38